custom_components/myskoda/coordinator.py

```diff
@@ -127,11 +127,11 @@ async def _on_charging_event(self, event: EventCharging):
         else:
             status = vehicle.charging.status
 
-            status.battery.remaining_cruising_range_in_meters = data.charged_range
+            status.battery.remaining_cruising_range_in_meters = (
+                data.charged_range * 1000
+            )
             status.battery.state_of_charge_in_percent = data.soc
             status.state = data.state
-            status.state = data.state
-        self.async_set_updated_data
```

Comment:
Why not set the update?

Reply:
This doesn't do anything, it's just the name of the method, without actually calling it. It think it's leftover from some refactoring.